assets/js/modules/adsense/datastore/index.js

```diff
@@ -32,6 +32,7 @@ import settings from './settings';
 import adblocker from './adblocker';
 import service from './service';
 import sites from './sites';
+import adBlockerRecovery from './ad-blocking-recovery';
```

Comment:
Inconsistent naming here: "adblocker" or "adblocking"? Let's align on one and use it consistently 👍